### PR TITLE
Add revive action to workflows

### PR DESCRIPTION
**Pull request by JanHoefelmeyer**
What:

Add revive action to github actions as described in https://github.com/marketplace/actions/revive-action 

Why:

Revive can catch potential mistakes and poorly written code. Solves  https://github.com/csaf-poc/csaf_distribution/issues/353

### CODE REVIEW

**Review by s-l-teichmann — APPROVED** [review]
LGTM

**Review by tschmidtb51 — APPROVED** [review]
In general, this looks good to me; but I haven't seen the output yet (obviously, as this GH Action was not triggered since no Go file was changed). So I suggest to merge this but leave #353 open until the next release to double check whether this action need some tweaking...  (so that we see results in a way that helps us spot mistakes easy. This might also end up in a case that we need an extra action for this.)

tl;dr: merge now, check impact in/on next PRs
